### PR TITLE
Morningstar Data

### PR DESCRIPTION
**Pull request by dtemkin**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [x] added entry to docs/source/whatsnew/vLATEST.txt

### CODE REVIEW

File: pandas_datareader/tests/mstar/test_daily.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+from pandas_datareader._testing import skip_on_exception
+from pandas_datareader._utils import RemoteDataError
+from pandas_datareader.data import MorningstarDailyReader
+
+
+class TestMorningstarDaily(object):
+
+    @classmethod
+    def setup_class(cls):
+        pytest.skip('Skip all Morningstar tests.')
```
Comment (bashtage) [inline pandas_datareader/tests/mstar/test_daily.py]: This is skipping all test.  They need to run.

File: pandas_datareader/tests/mstar/test_financials.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+from pandas_datareader._utils import RemoteDataError
+from pandas_datareader.data import BalanceSheetReader, \
+    CashflowStatementReader, IncomeStatementReader, KeyRatiosReader
+
+
+class TestMorningstarFinancials(object):
+
+    @classmethod
+    def setup_class(cls):
+        pytest.skip('Skip all Morningstar tests.')
```
Comment (bashtage) [inline pandas_datareader/tests/mstar/test_financials.py]: This is skipping all test.  They need to run.

**Comment by bashtage** [comment]
I've worked on this quite a bit on my branch.  This is a nice addition but I think it is unfortunately far from ready. The daily reader is a very useful contribution.

**Comment by dtemkin** [comment]
I saw that you had made changes and I really appreciate the effort. The financials data was beyond tricky because of the way it was formatted. I was having the same feeling that it wasn't ready especially given that I could not figure out how to parse the Growth Key Ratios. Do you think I should I remove the `financials` component and re-commit? Or were you referring to the daily OHLCV data as well?

**Comment by bashtage** [comment]
It might help to split the financials from the OHLCV.  The OHLCV reader likely has a much larger audience.

OHLCV passes all tests in my branch.  The financials still has a few issues.

**Comment by bashtage** [comment]
Thanks for this -- I cleaned it a bit and it was committed today.

**Pull request closed without merging**